n6001/hardware/ofs_n6001/build/scripts/run.sh

```diff
@@ -103,72 +108,20 @@ if [ -n "$OFS_ASP_ENV_ENABLE_ASE" ]; then
     exit $?
 fi
 
-echo "Calling compile_script"
-quartus_sh -t scripts/compile_script.tcl "$BSP_FLOW"
-FLOW_SUCCESS=$?
-
-# Report Timing
-# =============
-DO_ADJUST_PLLS=1
-PLL_METADATA_FILE="pll_metadata.txt"
-if [ $FLOW_SUCCESS -eq 0 ]
-then
-    if [ $DO_ADJUST_PLLS -eq 0 ]; then
-        echo "Not running adjust_plls.tcl. We still need to create a pll_metadata.txt file. Doing that now with 1x clock @ 150MHz and 2x clock @ 300 MHz."
-        echo "clock-frequency-low:150 clock-frequency-high:300" >> "$PLL_METADATA_FILE"
-    else
-        echo "Running adjust_plls.txt script to find the highest valid kernel_clock."
-        quartus_sh -t scripts/adjust_plls.tcl ofs_top "${BSP_FLOW}"
-    fi
-else
-    echo "ERROR: kernel compilation failed. Please see quartus_sh_compile.log for more information."
-    exit 1
-fi
-
-#run packager tool to create GBS
-BBS_ID_FILE="fme-ifc-id.txt"
-if [ -f "$BBS_ID_FILE" ]; then
-    FME_IFC_ID=$(cat $BBS_ID_FILE)
-    echo "FME_IFC_ID is $FME_IFC_ID"
-    cat $BBS_ID_FILE
-else
-    echo "ERROR: fme id not found."
-    exit 1
-fi
+echo "Starting: quartus_sh --flow compile $Q_REVISION -c $BSP_FLOW"
```

Comment:
this gets rid of compile-script, too. This simple command will also result in the gen-gbs.tcl script being executed (from pr-build-template)